src/core/thread/data_poll_manager.cpp

```diff
@@ -201,6 +201,8 @@ void DataPollManager::HandlePollSent(ThreadError aError)
             shouldRecalculatePollPeriod = true;
         }
 
+        otLogInfoMac(GetInstance(), "Sent data poll");
```

Comment:
Did not remember the other one...
Let's keep both. We change the one from `SendDataPoll()` to `"Sending poll"` and keep at `DEBG` level. The new one says "Sent poll".